src/Pokemon.java

```diff
@@ -0,0 +1,109 @@
+class Pokemon {
+
+    private String type;
+    protected static String name;
+    protected int hp;
+    protected String level;
+    private String specialAttack;
+    private String move;
+
+
+    public Pokemon (String type, String name, int hp, String level, String specialAttack, String move) {
+        this.type = type;
+        this.name = name;
+        this.hp = hp;
+        this.level = level;
+        this.specialAttack = specialAttack(specialAttack);
+        this.move = moves(move);
```

Comment:
De `moves()` methode is hier niet handig. Je had hier beter `this.moves=moves` kunnen doen. Het `move` attribuut wordt namelijk ook gebruikt in de `moves()` methode.